packages/fetchai/skills/simple_service_registration/behaviours.py

```diff
@@ -21,6 +21,7 @@
 
 from typing import Any, Optional, cast
 
+from aea.helpers.search.models import Description
```

Comment:
This also has a matching section in the docs, so update it there (https://docs.fetch.ai/aea/skill-guide/)